MediaGalleryIntegration/modman

```diff
@@ -0,0 +1 @@
+./ app/code/Magento/MediaGalleryIntegration
```

Comment:
Don't think we need to include this file

Reply:
we need to do this to be able to install this module using modman